### ISSUE TITLE
Bug: Incomplete diff when the file contains a long line of Unicode characters and spaces

### ISSUE THREAD

**Issue by ouuan** [jonas/tig#1080]
### Preparation

```bash
$ git init
$ echo "a" > test
$ git add .
$ git commit -m "Initial commit"
$ echo "。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。 ，，，，，，，，，，，，，，，，，，，，，，，，，，，，，，，，，，，，，，，，， 。。。。。。。。。。。" > test
$ git commit -am "Update"
```

### The output of tig

```bash
$ tig show
```

![screenshot](https://user-images.githubusercontent.com/30581822/106285607-a5628d00-627f-11eb-803e-88bdacec3e72.png)

### The expected output

```bash
$ git show
```

![screenshot](https://user-images.githubusercontent.com/30581822/106285698-bdd2a780-627f-11eb-993f-4c38a13fa45f.png)

## Environment

- tig: 2.5.1
- git: 2.30.0
- ncursesw: 6.2.20200212
- readline: 8.1
- OS: Arch Linux
- Shell: zsh 5.8
- Terminal: yakuake

```bash
$ tput cols  
169
$ tput lines 
46
```

**Comment by koutcher** [comment]
This is fixed in master by 484aa21.

**Comment by ouuan** [comment]
> This is fixed in master by [484aa21](https://github.com/jonas/tig/commit/484aa21a6f9ec6090a78b111eb429f9d922e25b9).

Tested just now. Hope there will be a new release soon.
